CMakeLists.txt

```diff
@@ -58,6 +58,11 @@ function(enableSanitizer SANITIZER)
     set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -fsanitize=${SANITIZER}" PARENT_SCOPE)
 endfunction()
 
+if(KVS_DEFAULT_STACK_SIZE)
```

Comment:
Can we add this to the top of the cmake list file as a valid option: 

`option(KVS_DEFAULT_STACK_SIZE "Setting thread default stack size in bytes" OFF)`

Reply:
That would give it a default value of OFF woudln't it?

Reply:
Yeah. 
The reason for the ask was to provide easy visibility for anyone looking into CMake file for a list of options. But, if we have documentation on how to set this up, I am fine not having this.